README.md

```diff
@@ -1,5 +1,12 @@
 # BCRegistry
 
+This project requires node v14.x.x (npm v6.14.x). You can use Node Version manager (nvm) to manage several versions of node...
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+nvm install 14
```

Comment:
can you update this to 15.14.0? Thanks for adding the documentation :)

Reply:
Didn't Node 15 have security issues? Anyway, it's not listed as an official release. Should we be using v16 instead?

https://nodejs.org/en/about/releases/

Reply:
Actually, Node 15 is an odd number and will not have Long Term Support. Node 15 is already End Of Life. We should be using Node 16.

https://nextlinklabs.com/insights/node-15-vs-16

Reply:
I don't 15 had the same security issues as ppr, but it is required for ppr currently because of a fas pkg we need. I guess for this it doesn't really matter as long as it works locally on whatever you run. Not sure what the deployment uses currently to generate it but that should be the one we put in the docs

Reply:
ah you are right though we will need to get that upgraded

Reply:
The CI action uses Node 14 ("Resolved as '14.18.2'").

I see you changed the lockfile to version 2 (presumably using Node 15). And I just checked in an updated lockfile, also version 2 (using Node 16), and the latest CI actions ran fine, so I guess it doesn't care.

**So then, are we telling devs to use Node 16 with this project?**

Reply:
sure why not

Reply:
OK, please check the new README file and resolve this conversation. Thx!

Reply:
^^ This will be updated in the next PR (so you guys don't have to keep checking THIS PR for changes).